testbed/tests/scenarios.go

```diff
@@ -190,6 +195,89 @@ func Scenario10kItemsPerSecond(
 	tc.ValidateData()
 }
 
+func Scenario10kScrapeItemsPerSecond(
+	t *testing.T,
+	sender testbed.DataSender,
+	receiver testbed.DataReceiver,
+	resourceSpec testbed.ResourceSpec,
+	resultsSummary testbed.TestResultsSummary,
+	processors map[string]string,
+	extensions map[string]string,
+	scrapeInterval time.Duration,
+) {
+	resultDir, err := filepath.Abs(path.Join("results", t.Name()))
+	require.NoError(t, err)
+
+	// Because there are 7 datapoints per metric, this amounts to 10k items per second
```

Comment:
Where does the number 7 come from? If it is elsewhere how do we make sure the 2 places stay in sync?